app/bundles/ApiBundle/Form/Type/ClientType.php

```diff
@@ -203,12 +203,4 @@ public function configureOptions(OptionsResolver $resolver)
             ]
         );
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getBlockPrefix()
```

Comment:
We left these methods intentionally. Although they are not used by Symfony anymore they were necessary to keep some of JS still working. We definitely don't need all of them but I'm unsure which are needed. And we don't have any end-to-end tests that would point out which JS is broken. See https://github.com/mautic/mautic/blob/5.x/UPGRADE-3.0.md?plain=1#L472

Reply:
As I cannot find any place that this method is used really, let's remove it. We'll find out in the grand testing of the alpha version whether something is not working.